.github/workflows/check-link-rot.yaml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+
+name: check-link-rot
+
+jobs:
+  check-link-rot:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      R_KEEP_PKG_SOURCE: yes
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: "devel"
+          http-user-agent: "release"
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          pak-version: devel
+          extra-packages: |
+            any::rcmdcheck
+            any::urlchecker
+
+      - name: Run URL checker
+        run: |
+          options(crayon.enabled = TRUE)
+          rotten_links <- urlchecker::url_check(progress = FALSE)
+          print(rotten_links)
+          if (length(rotten_links$URL) > 0L) {
+            stop("Some URLs are outdated and need to be updated.", call. = FALSE)
```

Comment:
Will the concrete links and locations be visible?

Reply:
> Will the concrete links and locations be visible?

Yes, for a failed workflow example, see [this](https://github.com/IndrajeetPatil/minimalReprex/actions/runs/3434717367/jobs/5726296581) run. 

Also, for an example of a successful run, see [this](https://github.com/IndrajeetPatil/statsExpressions/actions/runs/3434847347/jobs/5726551334) one.

Reply:
Cool, thanks!